BAO/Configurations/clcommon__Asset_Class__c/BusinessCertificatesofDeposit.json

```diff
@@ -0,0 +1,15 @@
+{
+     "attributes": {
+          "type": "clcommon__Asset_Class__c"
+     },
+     "Name": "Business Certificates of Deposit",
+     "clcommon__Asset_Category__c": null,
+     "clcommon__Asset_Class_API_Name__c": null,
+     "clcommon__External_ID__c": "BusinessCertificatesofDeposit",
```

Comment:
Please follow the same naming convention we followed earlier. Replace space with underscore "_". Please do this for all the following records as well.
"